app/utils.js

```diff
@@ -0,0 +1,34 @@
+function slidePlayListIn(elem) {
```

Comment:
funcţiile de animaţii ar trebui regândite

foloseşte animaţii css pe care le controlezi prin clase ( slide-in vs slide-out  de exemplu ) şi ascultă după evenimentul de animationEnd

nu uita să cureţi listenerul după animationEnd